### PR TITLE
Wallet UI changes (direct token-based authentication)

### PR DESCRIPTION
**Pull request by mmjee**
# Pull Request Template

Resolves or continues [AR-6206](https://team-1624093970686.atlassian.net/browse/AR-6206).

## Blocking dependencies

None.

## Changes

See ticket and and the Auth SDK changes.

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

[AR-6206]: https://team-1624093970686.atlassian.net/browse/AR-6206?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/pages/backCompat/signIn.vue [hunk 1]
```diff
@@ -52,7 +52,9 @@ const penpalMethods = {
   isLoginAvailable: (kind: SocialLoginType) =>
     availableLogins.value.includes(kind),
   getPublicKey: handleGetPublicKey,
-  getAvailableLogins: () => [...availableLogins.value],
+  getAvailableLogins: () => [
+    ...availableLogins.value.filter((item) => item !== 'firebase'),
```
Comment (mmjee) [inline src/pages/backCompat/signIn.vue:56]: Why don't you directly return the filtered array?

Comment (mmjee) [inline src/pages/backCompat/signIn.vue:56]: Why don't you directly return the filtered array?

Comment (karthik-durai) [reply src/pages/backCompat/signIn.vue:56]: sure, will do that

File: src/pages/signInV2.vue [hunk 2]
```diff
@@ -251,6 +260,52 @@ async function handleSubmit() {
...
+        token: string
+      }
+
+      const info = await kr.getPrivateKey({
+        verifier: BearerAuthentication.firebase,
+        id: data.uid,
+        idToken: data.token,
+      })
+
+      // TODO
```
Comment (shrinathprabhu) [inline src/pages/signInV2.vue:290]: What is left here? @mmjee

Comment (mmjee) [reply src/pages/signInV2.vue:290]: Error handling, treating Firebase differently. This is probably the hackiest feature we have.

**Review by shrinathprabhu — APPROVED** [review]

**Pull request merged**